src/debugger.ts

```diff
@@ -135,7 +135,12 @@ function searchForCompilerPathInCache(cache: CMakeCache): string|null {
   return null;
 }
 
-export async function getDebugConfigurationFromCache(cache: CMakeCache, target: ExecutableTarget, platform: string, debuggerPathOverride?: string):
+export enum miModes {
```

Comment:
A type is added here, but I don't think we ever validate the user setting.  It's fine for now, but we should open a task to to validate everything that comes in as an enum to ensure the value is valid.  VS Code will squiggle, but I think we can still get any string value here.

Also, the type should start with capital letters per our coding conventions.  `MIModes`

Reply:
Good idea, opened work item https://github.com/microsoft/vscode-cmake-tools/issues/1741.